### PR TITLE
Shortcuts refactor & MacOS tooltips

### PR DESCRIPTION
**Pull request by Alotor**

### CODE REVIEW

File: frontend/resources/locales.json [hunk 1]
```diff
@@ -3823,166 +3834,172 @@
...
+    "used-in" : [ "src/app/main/ui/workspace/context_menu.cljs:184", "src/app/main/ui/workspace/context_menu.cljs:196", "src/app/main/ui/workspace/sidebar/options/component.cljs:97", "src/app/main/ui/workspace/sidebar/options/component.cljs:103" ],
     "translations" : {
       "en" : "Update master component",
       "es" : "Actualizar componente maestro"
     }
   },
+  "workspace.sidebar.history" : {
+    "used-in" : [ "src/app/main/ui/workspace/left_toolbar.cljs:122" ],
+    "translations" : {
+      "en" : "History (%s)"
```
Comment (hirunatan) [inline frontend/resources/locales.json]: I think for new texts we should at least give the spanish translation, which is already complete.

File: frontend/src/app/main/ui/viewer/header.cljs [hunk 2]
```diff
@@ -273,7 +274,7 @@
         :on-zoom-to-200 (st/emitf dv/zoom-to-200)}]
 
       [:span.btn-icon-dark.btn-small.tooltip.tooltip-bottom
-       {:alt (t locale "viewer.header.fullscreen")
+       {:alt (t locale "viewer.header.fullscreen" (sc/get-tooltip :fullscreen))
```
Comment (hirunatan) [inline frontend/src/app/main/ui/viewer/header.cljs]: There is no shortcut for fullscreen in data/viewer/shortcuts.cljs

File: frontend/src/app/main/data/viewer/shortcuts.cljs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+   :select-all    {:tooltip (ds/meta "A")
+                   :command (ds/c-mod "a")
+                   :fn (st/emitf (dv/select-all))}
+
+   :zoom-50       {:tooltip (ds/shift "0")
+                   :command "shift+0"
+                   :fn (st/emitf dv/zoom-to-50)}
+
+   :reset-zoom    {:tooltip (ds/shift "0")
```
Comment (hirunatan) [inline frontend/src/app/main/data/viewer/shortcuts.cljs]: This should be "1"

File: frontend/src/app/main/data/workspace.cljs [hunk 4]
```diff
@@ -1223,20 +1222,26 @@
...
-  [{:keys [team-id] :as project}]
-  (ptk/reify ::go-to-viewer
-    ptk/WatchEvent
-    (watch [_ state stream]
-      (rx/of ::dwp/force-persist
-             (rt/nav :dashboard-projects {:team-id team-id})))))
+  ([] (rx/of ::dwp/force-persist
+             (rt/nav :dashboard-projects)))
+  ([{:keys [team-id] :as project}]
+   (ptk/reify ::go-to-viewer
```
Comment (hirunatan) [inline frontend/src/app/main/data/workspace.cljs]: This should be ::go-to-dashboard

File: frontend/src/app/main/ui/viewer/header.cljs [hunk 5]
```diff
@@ -273,7 +274,7 @@
         :on-zoom-to-200 (st/emitf dv/zoom-to-200)}]
 
       [:span.btn-icon-dark.btn-small.tooltip.tooltip-bottom
```
Comment (hirunatan) [inline frontend/src/app/main/ui/viewer/header.cljs]: Thils should be .tooltip-bottom-left, for it not to go offscreen.

File: frontend/src/app/main/ui/workspace/header.cljs [hunk 6]
```diff
@@ -273,7 +276,7 @@
         :on-zoom-selected #(st/emit! dw/zoom-to-selected-shape)}]
 
       [:a.btn-icon-dark.btn-small.tooltip.tooltip-bottom
```
Comment (hirunatan) [inline frontend/src/app/main/ui/workspace/header.cljs]: Thils should be .tooltip-bottom-left, for it not to go offscreen.

File: frontend/src/app/main/data/workspace.cljs [hunk 7]
```diff
@@ -1223,20 +1222,26 @@
...
 
 (defn go-to-dashboard
-  [{:keys [team-id] :as project}]
-  (ptk/reify ::go-to-viewer
-    ptk/WatchEvent
-    (watch [_ state stream]
-      (rx/of ::dwp/force-persist
-             (rt/nav :dashboard-projects {:team-id team-id})))))
+  ([] (rx/of ::dwp/force-persist
+             (rt/nav :dashboard-projects)))
```
Comment (hirunatan) [inline frontend/src/app/main/data/workspace.cljs]: This is not working (does nothing when pressed G D)

File: frontend/resources/locales.json (hunk 1 above)
Comment (Alotor) [reply frontend/resources/locales.json]: OK

File: frontend/src/app/main/ui/viewer/header.cljs (hunk 2 above)
Comment (Alotor) [reply frontend/src/app/main/ui/viewer/header.cljs]: Yeah, it's something I was thinking to add but wasn't so straightforward as I thought. I'll remove the reference

File: frontend/src/app/main/data/viewer/shortcuts.cljs (hunk 3 above)
Comment (Alotor) [reply frontend/src/app/main/data/viewer/shortcuts.cljs]: Ups! nice catch

File: frontend/src/app/main/data/workspace.cljs (hunk 4 above)
Comment (Alotor) [reply frontend/src/app/main/data/workspace.cljs]: OK

File: frontend/src/app/main/ui/workspace/header.cljs (hunk 6 above)
Comment (Alotor) [reply frontend/src/app/main/ui/workspace/header.cljs]: OK

**Pull request merged**